Foundation/Data.swift

```diff
@@ -993,8 +1011,8 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
     public typealias Index = Int
     public typealias Indices = Range<Int>
     
-    @_versioned internal var _backing : _DataStorage
-    @_versioned internal var _sliceRange: Range<Index>
+    @usableFromInline internal var _backing : _DataStorage
+    @usableFromInline internal var _sliceRange: Range<Index>
```

Comment:
Is it now okay to bring it in?

https://github.com/apple/swift-corelibs-foundation/pull/1502